New Text Document.txt

```diff
@@ -2,10 +2,10 @@
             /                                                    \
            |    _____________________________________________     |
            |   |                                             |    |
-           |   |  C:\User\Dude> java -verson                 |    |
-           |   |                                             |    |
-           |   |                                             |    |
-           |   |                                             |    |
+           |   |  C:\User\Dude> java -version                |    |
+           |   |  C:\User\Dude> echo $truth                  |    |
+           |   |  C# is Java but better.                     |    |
```

Comment:
This will surely be a spicy file for future students to discover. Good thinking.